src/views/Swap/components/SwapWarningModal/Acknowledgement.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'contexts/Localization'
+import { Text, Flex, Checkbox, Button } from '@pancakeswap/uikit'
+
+interface AcknowledgementProps {
+  handleContinueClick: () => void
+}
+
+const Acknowledgement: React.FC<AcknowledgementProps> = ({ handleContinueClick }) => {
```

Comment:
Generic Acknowledgement component that can easily be plugged into any token warning